site/documentation/content/admin-guide/monitoring-tracing-config.md

```diff
@@ -113,20 +113,28 @@ to generate, emit, collect, process and export tracing data.
 | `QUARKUS_OPENTELEMETRY_TRACER_SAMPLER_RATIO`<br>`quarkus.opentelemetry.tracer.sampler.ratio` | *double* | | The ratio of traces to be exported. To be used if the sampler is set to `ratio`. Must be within `[0.0, 1.0]`. Note that in a YAML configuration, the `ratio` sampler has to be configured [using `~` as key](https://quarkus.io/guides/config-yaml#configuration-key-conflicts), in order to set the ratio value as well. Example:<br><code>sampler:<br>&nbsp;&nbsp;~: ratio<br>&nbsp;&nbsp;ratio: 0.8</code> |
-| `QUARKUS_OPENTELEMETRY_TRACER_EXPORTER_OTLP_ENDPOINT`<br>`quarkus.opentelemetry.tracer.exporter.otlp.endpoint` | *string* | | The OTLP endpoint of the OpenTelemetry Collector to connect to. The endpoint must start with either `http://` or `https://`. |
+| `QUARKUS_OPENTELEMETRY_TRACER_EXPORTER_OTLP_ENDPOINT`<br>`quarkus.opentelemetry.tracer.exporter.otlp.endpoint` | *string* | `-` | The OTLP endpoint of the OpenTelemetry Collector to connect to. The endpoint must start with either `http://` or `https://`. |
 
 Please refer to the
 [Quarkus OpenTelemetry documentation](https://quarkus.io/guides/opentelemetry#quarkus-opentelemetry-exporter-otlp_configuration)
-for further configuration trace exporter options.
+for further configuration options for the tracer exporter.
+
+Alternatively, the sampler can be configured using the following
+[OpenTelemetry SDK](https://opentelemetry.io/docs/reference/specification/sdk-environment-variables/#general-sdk-configuration)
+configuration properties. These properties, if set, take precendence over the corresponding Quarkus specific properties.
+
+| OS Environment Variable<br>Java System Property | Type      | Default Value | Description  |
+| :---------------------------------------------- | :-------- | :------------ | :------------|
+| `OTEL_TRACER_SAMPLER`<br>`otel.tracer.sampler`        | *string*  | `always_on`     | The sampler to use for determining if a tracing span should be reported. Please refer to the [OpenTelemetry documentation](https://opentelemetry.io/docs/reference/specification/sdk-environment-variables/#general-sdk-configuration) for supported values.<br>In addition to the standard samplers, a rate-limiting sampler can be configured by setting this property to `rate_limiting` or `rate-limiting` and using the `OTEL_TRACES_SAMPLER_ARG` property to set the maximum number of traces to report per second. |
```

Comment:
A note could be added that there will always be a parent-based sampler in front of the configured sampler, meaning there is no need to configure a `parentbased_*` sampler.